targets/frontend/src/components/themes/List.js

```diff
@@ -0,0 +1,147 @@
+/** @jsx jsx  */
+
+import Link from "next/link";
+import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
+import { IoIosArrowDroprightCircle, IoIosReorder } from "react-icons/io";
+import {
+  SortableContainer,
+  SortableElement,
+  SortableHandle,
+} from "react-sortable-hoc";
+import { IconButton } from "src/components/button";
+import { useUser } from "src/hooks/useUser";
+import { Alert, Card, Flex, jsx, Text } from "theme-ui";
+
+const formatRelationsIntoThemes = (relations = []) =>
+  relations
+    .sort(({ position: a }, { position: b }) => a - b)
+    .map(({ id, child }) => ({
+      relationId: id,
+      ...child,
+    }));
+
+const List = ({ relations, updateThemesPosition }) => {
```

Comment:
I did call it child at first but it was confusing because we also need to keep the ID of the relation connecting to its parent. So I find it more clear to specify here that it is a relation (with its own id) and as you unfold it, call it child once we indeed reach the child (in the next map iteration)